lib/routes.js

```diff
@@ -64,6 +76,7 @@ function fastifySwagger (fastify, opts, done) {
     url: '/uiConfig',
     method: 'GET',
     schema: { hide: true },
+    ...hooks,
```

Comment:
I do not think the UI can works properly when fetching the config require hooks or authentication as the script do not bring the `authorization` header.

Reply:
It does

![image](https://user-images.githubusercontent.com/11404065/133399965-9119a084-e09a-4f6d-b090-d4d3038d4ab0.png)

Fetch includes the authorization for the same-origin
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#sending_a_request_with_credentials_included

Reply:
Oh. I didn't imagine it does.